pkg/services/cleanup/cleanup.go

```diff
@@ -184,7 +188,7 @@ func (srv *CleanUpService) deleteOldLoginAttempts(ctx context.Context) {
 	cmd := models.DeleteOldLoginAttemptsCommand{
 		OlderThan: time.Now().Add(time.Minute * -10),
 	}
```

Comment:
QQ: This kind of cleanup job should eventually move over to the service, right?

Do we want a centralized cleanup service that cleanup jobs register with or do we want to register a whole lot of different but similar cleanup jobs as background jobs? I think having a single service similar to usage stats would make sense?